src/createXcmTypes/util/getAssetId.spec.ts

```diff
@@ -58,14 +58,14 @@ describe('getAssetId', () => {
 		it('Should correctly return the xcAsset multilocation when given a valid asset symbol', async () => {
 			const expected = '{"v1":{"parents":1,"interior":{"x2":[{"parachain":2023},{"palletInstance":10}]}}}';
 
-			const result = await getAssetId(bifrostAssetsApi._api, registry, 'movr', 'bifrost', 2, false);
+			const result = await getAssetId(bifrostAssetsApi.api, registry, 'movr', 'bifrost', 2, false);
 
 			expect(result).toEqual(expected);
 		});
 
 		it('Should correctly error when given an invalid xcAsset symbol', async () => {
 			await expect(async () => {
-				await getAssetId(bifrostAssetsApi._api, registry, 'TEST', 'bifrost', 2, true);
+				await getAssetId(bifrostAssetsApi.api, registry, 'TEST', 'bifrost', 2, true);
 			}).rejects.toThrowError(`parachain assetId TEST is not a valid symbol assetId in bifrost`);
```

Comment:
```suggestion
			}).rejects.toThrow(`parachain assetId TEST is not a valid symbol assetId in bifrost`);
```
There are more instances but I prefer not to mention everything one by one in case this is not a change you would like to implement.